service/frontend/workflowHandler.go

```diff
@@ -410,19 +410,39 @@ func (wh *WorkflowHandler) UpdateDomain(
 			return nil, err
 		}
 	} else {
+		wh.GetLogger().Info("Failover request received, checking failover permission.",
```

Comment:
... and error log for all the failures.

Reply:
I would add another log after 
```
resp, err := wh.domainHandler.UpdateDomain(ctx, updateRequest)
```
if it fails it would say DomainUpdate was successful or DomainUpdate failed with error logged.